.github/workflows/nightly-replay-verify.yaml

```diff
@@ -0,0 +1,37 @@
+name: "replay-LLT-transactions-nightly"
+on:
+  # Allow triggering manually
+  workflow_dispatch:
+    inputs:
+      GIT_SHA:
+        required: false
+        type: string
+        description: The git SHA1 to test. If not specified, Forge will check the latest commits on the current branch
+  schedule:
+    - cron: "0 * * * */1"
+  pull_request:
+    paths:
+      - '.github/workflows/nightly-replay-verify.yaml'
+
+env:
+  BUCKET: ${{ secrets.LLT_BACKUP_BUCKET }}
+  SUB_DIR: e1
+
+jobs:
+  replay-LTT-transactions:
+    timeout-minutes: 720
+    runs-on: high-perf-docker
```

Comment:
Slightly concerned about the cost of running this job every day for 8h on such a fat instance type.
The high-perf-docker instances are 60CPU + 240GB Memory (c2-standard-60 specifically: https://cloud.google.com/compute/docs/compute-optimized-machines#c2_machine_types) and are mainly designed to run expensive rust compilation steps. If that job actually can leverage 64CPU for the 8h that's fine with me but it looks like it's rather using ~5 cores only:
<img width="1560" alt="image" src="https://user-images.githubusercontent.com/1221897/194465580-c9aa4094-d750-4395-9f5e-a62ef54b01b2.png">
https://console.cloud.google.com/kubernetes/pod/us-west1-b/ci-cluster/actions-runner/high-perf-docker-xjdfq-b997x/details?authuser=1&project=aptos-ci

Wondering if we can tune some stuff to make it use more cores / be faster or use a different instance type?

Reply:
yea looking at the CLI, it looks like replay concurrency can be tuned. @msmouse @runtian-zhou would you know if the defaults (which seem to default to number of cores available) would suffice?